components/announcement_bar/default_announcement_bar/announcement_bar.jsx

```diff
@@ -98,7 +114,7 @@ export default class AnnouncementBar extends React.PureComponent {
                     style={linkStyle}
                     onClick={this.handleClose}
                 >
-                    {'×'}
+                    {this.props.isTallBanner ? '' : '×'}
```

Comment:
Recommend using unicode character here as opposed this -- `'\u00E9'` etc.

Reply:
Good call